docs/release-notes.md

```diff
@@ -77,8 +77,28 @@ UNRELEASED
   of your stbt config file, and let us know on the mailing list as we may
   remove the workaround completely in a future release.
 
+* A call to `stbt.get_frame()` is no-longer guaranteed to return a new frame, it
+  may return the same frame that the previous call to `stbt.get_frame()`
+  returned. This may have subtle effects on the timing of existing test-scripts.
+  Functions that depend on this behaviour should be refactored to use the
+  `stbt.frames()` iterator method instead.
```

Comment:
"no longer blocks until a new frame is available."

The benefits are that you can call `stbt.get_frame` from multiple threads.

Reply:
Fixed.